src/aura/Work_Items_List/Work_Items_ListController.js

```diff
@@ -0,0 +1,28 @@
+({
+	doInit : function(component, event, helper) {
+		var statusType = component.get('v.workItemsStatus');
+		var contact = component.get('v.contactRecordId');
+		
+		var action;
+		console.log("holaaa " + contact);
+		if(contact !== undefined && contact.length >= 0){
+			action = component.get('c.getWorkItemsByContactByStatusInLast30Days');
+			action.setParam('contact', contact);
+		}else{
+			action = component.get('c.getWorkItemsByStatusInLast30Days');
+		}
+		action.setParam('status', statusType);
+
+		action.setCallback(this, function(response){
+			var state = response.getState();
+			if(state == 'SUCCESS'){
```

Comment:
Using strict equal and strict not equal instead of the other versions helps to maintain data type integrity through the code.